File: 프로그래머스/0/120821. 배열 뒤집기/배열 뒤집기.java
class Solution {
    public int[] solution(int[] num_list) {
        int tmp = 0;
        
        for(int i = 0; i < (num_list.length / 2); i++) {
            tmp = num_list[num_list.length - 1 - i];
            num_list[num_list.length - 1 - i] = num_list[i];
            num_list[i] = tmp;
        }
        return num_list;
    }
}

File: 프로그래머스/0/120821. 배열 뒤집기/배열 뒤집기.go
func solution(num_list []int) []int {
    tmp := 0
        
    for i := 0; i < (len(num_list) / 2); i++ {
        tmp = num_list[len(num_list) - 1 - i]
        num_list[len(num_list) - 1 - i] = num_list[i]
        num_list[i] = tmp
    }
    
    return num_list
}